### PR TITLE
[ko] Translate /content/ko/training/_index.html

### PR DESCRIPTION
**Pull request by bricksky**
The relationship between subject and predicate was not natural.
To make it easier for users to understand, we modified the relationship between subject and predicate.

### CODE REVIEW

**Comment by seokho-son** [comment]
Hi @bricksky 

Thank you for the suggestion ! :)

I think the original sentence is natural and looks aligned with the English source.

공인 쿠버네티스 애플리케이션 개발자 시험은 **사용자가 쿠버네티스의 클라우드 네이티브 애플리케이션을 디자인, 구축, 구성과 노출을 할 수 있음**을 인증합니다.

The Certified Kubernetes Application Developer exam certifies **that users can design, build, configure, and expose cloud native applications for Kubernetes**.

공인 쿠버네티스 애플리케이션 개발자 시험은 X을 인증합니다.
if X = `사용자가 쿠버네티스의 클라우드 네이티브 애플리케이션을 디자인, 구축, 구성과 노출을 할 수 있음`

What do you think so ?

---

By the way, You can use Korean for comments if you prefer. :)

**Comment by seokho-son** [comment]
I think `사용자가` can be removed from the sentence to minimize misunderstanding.

**Comment by seokho-son** [comment]
PTAL @bricksky :)

**Comment by jmyung** [comment]
> I think `사용자가` can be removed from the sentence to minimize misunderstanding.
agreed.

Thank you @bricksky 👍

File: content/ko/training/_index.html [hunk 1]
```diff
@@ -97,7 +97,7 @@ <h5>
           <h5>
             <b>공인 쿠버네티스 애플리케이션 개발자(Certified Kubernetes Application Developer, CKAD)</b>
           </h5>
-          <p>공인 쿠버네티스 애플리케이션 개발자 시험은 사용자가 쿠버네티스의 클라우드 네이티브 애플리케이션을 디자인, 구축, 구성과 노출을 할 수 있음을 인증합니다.</p>
+          <p>공인 쿠버네티스 애플리케이션 개발자 시험은 사용자에게 쿠버네티스의 클라우드 네이티브 애플리케이션을 디자인, 구축, 구성과 노출을 할 수 있음을 인증합니다.</p>
```
Comment (seokho-son) [inline content/ko/training/_index.html:100]:
```suggestion
          <p>공인 쿠버네티스 애플리케이션 개발자 시험은 쿠버네티스의 클라우드 네이티브 애플리케이션을 디자인, 빌드, 구성, 노출(expose)할 수 있음을 인증합니다.</p>
```

**Review by seokho-son — COMMENTED** [review]
This is my suggestion. :)

**Comment by seokho-son** [comment]
Hi @bricksky,
It appears that this PR has become stale. Based on the decision from the SIG Docs localization subproject, if a PR author does not provide feedback to reviewers for 3 months, localization leads will assume the PR is stale and close it.
Please feel free to reopen this issue if you are willing to continue with it.

/close

**Pull request closed without merging**